### PR TITLE
Add option of creating initial super user

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@miltontony @JayH5 ready for review

**Comment by JayH5** [comment]
@smn: can you add a note somewhere in the README, please?

File: README.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ Run [Django](https://www.djangoproject.com) projects from source using [Gunicorn
 #### Step 0: Get your Django project in shape
 There are a few ways that your Django project needs to be set up in order to be compatible with this Docker image.
 
-**setup.py**
```
Comment (JayH5) [inline README.md:10]: @smn: your text editor is tripping my formatting :cry:

Comment (smn) [inline README.md:10]: eh sorry! Is there special meaning to the trailing characters? It seems to be doing this automatically.

**Comment by miltontony** [comment]
👍 once formatting fixed

**Pull request merged**